main.go

```diff
@@ -119,14 +119,19 @@ func listPRs(client *github.Client) []*github.PullRequest {
 					Sort:      "updated",
 					Direction: "desc",
 					ListOptions: github.ListOptions{
-						Page: page,
+						Page:    page,
+						PerPage: 50,
 					},
 				})
 			})
 			if err != nil {
 				log.Fatalf("Unable to list PRs for page: %v: %v", page, err)
 			}
 			prs = append(prs, p...)
+			// Early exit
```

Comment:
Should this be after

```
if page == 0 {
    break
}
```
?

My reasoning being if there are no PRs, then the slice might be empty and this will go out of bounds.